mods/cnc/tilesets/desert.yaml

```diff
@@ -2202,35 +2202,30 @@ MultiBrushCollections:
 			Template: 14
 			Segment:
 				Start: Cliff.L
-				Inner: Cliff
 				End: Cliff.L
 				Points: 2,2, 1,2, 1,1, 0,1
 		MultiBrush@15:
 			Template: 15
 			Segment:
 				Start: Cliff.L
-				Inner: Cliff
 				End: Cliff.L
 				Points: 2,1, 1,1, 0,1
 		MultiBrush@16:
 			Template: 16
 			Segment:
 				Start: Cliff.L
-				Inner: Cliff
 				End: Cliff.L
 				Points: 2,1, 1,1, 0,1
 		MultiBrush@17:
 			Template: 17
 			Segment:
 				Start: Cliff.L
-				Inner: Cliff
 				End: Cliff.L
 				Points: 2,1, 1,1, 0,1
 		MultiBrush@18:
 			Template: 18
 			Segment:
 				Start: Cliff.L
-				Inner: Cliff
 				End: Cliff.L
 				Points: 2,1, 1,1, 1,2, 0,2
 		MultiBrush@19:
```

Comment:
The cliffs bellow still have inner defined. It's true in all CNC and RA tilesets

Reply:
These ones also have a start or end of "Clear". Inner is deliberately kept here so that the game doesn't assume that "Clear" is a valid Inner type.

Reply:
Ah right, the text looks quite simmilar so I missread